src/edu/csus/ecs/pc2/core/scoring/DefaultScoringAlgorithm.java

```diff
@@ -245,6 +247,72 @@ private ProblemSummaryInfo calcProblemScoreData(TreeMap<Run,Run> treeMap, IInter
         return problemSummaryInfo;
     }
 
+    /**
+     * Get the Point Scoring and Statistics information for one problem.
+     *
+     * @return pc2.ex.ProblemScoreData
```

Comment:
This `@return` refers to a class in a non-existent package (`pc2.ex`).

Reply:
Yikes.  That must have been WAY before my time.  (I fixed calcProblemScoreData's comment as well while I was at it.)
Next push.